### DISCUSSION TITLE
Adding a custom icon set, not all displaying

### DISCUSSION THREAD

**Discussion by bkrisler** [squidfunk/mkdocs-material#2283] [Q&A] [answered]
Hello,

I am trying to add in a custom icon set (https://github.com/bpmn-io/bpmn-font) and not all are displaying. Is there a way to view debug logs, etc. to try and figure out why?

For example:

This displays just fine:
`:bpmn-tools-hand-tool:`
(**_.icons/bpmn/tools/hand-tool.svg_**)

But this is empty:
`:bpmn-task:`
(**_.icons/bpmn/task.svg_**)

Same for this:
`:bpmn-connection-multi:`
(**_.icons/bpmn/connection-multi.svg_**)

It appears to be hit or miss on which icons will display or not. Any suggestions on how to figure this out?

More info. I have:

```
| Tool                    | Name           | Description |
| ----------------------- | -------------- | ----------------------- |
| :bpmn-tools-hand-tool:  | Grab           | Activate the Hand tool  |
| :bpmn-tools-lasso-tool: | Lasso          | Activate the Lasso tool | 
| :bpmn-tools-space-tool: | Space          | Activate the create/remove space tool |
| :bpmn-connection-multi: | Global Connect | Activate the global connect tool |
| :bpmn-events-start-event-none: | Start Event | BPMN start event |
| :bpmn-events-end-event-none: | End Event | BPMN end event |
```

![image](https://user-images.githubusercontent.com/2789200/107234374-97380d80-69f1-11eb-8299-9dbb72e2629d.png)

Thanks

**Comment by bkrisler** [comment, marked as answer]
Found the issue. The images that were not displaying were all missing a *viewBox* attribute on the *svg* element. Hand added that and it worked.

**Reply by squidfunk** [reply]
Good to know, thanks for answering your own question! The images were likely shown, but due to the missing `viewbox` attribute they were cropped to their tiny upper left corners. I will add a small note to the docs to lookout for the `viewbox` attribute.
